pom.xml

```diff
@@ -327,6 +327,33 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
```

Comment:
Er det enforcing av en nyere versjon av pluginet? Hvorfor trenger vi det? (spør bare for å forstå - ikke fordi jeg er uenig :) )

Reply:
Copy pasta

Reply:
Er det for å tilby bedre støtte av Kotlin?